### PR TITLE
added audio mute functionality, polished user matching algorithm, add…

### LINKED ISSUE #56: Polish/Optimize User matching algorithm

### PR DESCRIPTION
**Pull request by lccov**
added audio mute functionality, polished user matching algorithm, add matches into database, and other minor changes

### CODE REVIEW

**Review by Rebecca-CYLi — APPROVED** [review]
LGTM

File: capstone-project/api/server.js [hunk 1]
```diff
@@ -57,31 +60,60 @@ io.on("connection", (socket) => {
...
+          db.query(`
+          INSERT INTO previously_matched (
+              user_1_id,
+              user_2_id
+          )
+          VALUES ($1, $2)
+          RETURNING id, user_1_id, user_2_id, match_timestamp;
+          `,
+          [user.user_id, peer.user_id], (err, res) => {
+            console.log(err ? err.stack : "Entered Successfully!".blue)
```
Comment (LifeBringer) [inline capstone-project/api/server.js]: Remove/comment out or use console.error or some alternative.

**Review by LifeBringer — CHANGES_REQUESTED** [review]
See comments

**Review by gabremo — APPROVED** [review]
Astonishing work, Lacie!!! 😲🤯

**Review by LifeBringer — APPROVED** [review]

**Pull request merged**